zio-cli/shared/src/main/scala/zio/cli/Command.scala

```diff
@@ -284,8 +284,9 @@ object Command {
         .flatMap {
           case CommandDirective.BuiltIn(BuiltInOption.ShowHelp(_, _)) =>
             helpDirectiveForChild orElse helpDirectiveForParent
-          case CommandDirective.BuiltIn(_) =>
+          case CommandDirective.BuiltIn(BuiltInOption.Wizard(_)) =>
```

Comment:
Fixing this, otherwise all built-in options except for `help` are treated as `wizard`